src/renderer/components/table/table.tsx

```diff
@@ -52,6 +53,16 @@ export const orderByUrlParam = createPageParam({
   isSystem: true,
 });
 
+export interface TableStorageSortParams {
+  tables: {
+    [tableId: string]: Partial<TableSortParams>;
+  }
+}
+
+const tableStorage = createStorage<TableStorageSortParams>("table_sort_params", {
+  tables: {}
+});
```

Comment:
I would extract all these logic to separated file as was done here:

https://github.com/lensapp/lens/pull/2392/files#diff-73214131a69d3577e42e23206c7f54b6c614b43693a0f443a8aa7071d704c811

Reply:
@ixrock yeah..does it make sense because you are going to get rid of `table` in 5.x ?

Reply:
I think it make sense because:
1. It will be available for use very soon right after `4.2.1`.
2. New table probably won't have save-to-storage features, so we will need it sooner or later.
3. It's almost done, why not merge it?

Reply:
@Nokel81 Since it's marked as `5.0.0` there's no need to merge the PR... But this changes could be moved to `4.2.1`. Wdyt?

Reply:
Given that we seem to be doing a slower 4.2.0 release. Sure, we might be able to get most of these things into 4.2.0. Will change the target.

Reply:
ok, thank you, I will move storage logic to a separated file